From 9b6950f76b6a8b526ec7d9bf012b9e7815d89143 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Mon, 06 May 2019 18:57:03 +0300
Subject: [PATCH] [SW-0] pll: lower wait time between clk switches

1 micro second should be more than enough for the clock switch to
stabilize and output the new frequency clock.
Lower the wait time to 1 micro second (instead of 20) between the
writes to clock switches.

Change-Id: Ic1d8bd6910fc6af98b17c397448dba4e1c19bc49
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/common/pll.c b/common/pll.c
index cf622ae..86c05c7 100644
--- a/common/pll.c
+++ b/common/pll.c
@@ -90,7 +90,7 @@
 			regmap_writel(HL_PLL_PLL_CLK, reg);
 		else { /* first change DIV_REF, then DIV_PLL */
 			regmap_writel(HL_PLL_DIV_REF, reg);
-			hl_pll_udelay(HL_PLL_MAX_WAIT_TIME);
+			hl_pll_udelay(1);
 			regmap_writel(HL_PLL_DIV_PLL, reg);
 		}
 	}
